src/osd/OSDMap.cc

```diff
@@ -4727,6 +4734,12 @@ int OSDMap::calc_pg_upmaps(
       }
       int osd = p->second;
       float deviation = p->first;
+      if (fast_aggressive && osd_to_skip.count(osd)) {
+	ldout(cct, 20) << " Skipping osd " << osd 
```

Comment:
Perhaps add a note  in the ldout that this is because of the `osd_calc_pg_upmaps_aggressively_fast` config option, just to make it explicitly clear why an osd is getting skipped.

Reply:
> Perhaps add a note in the ldout that this is because of the `osd_calc_pg_upmaps_aggressively_fast` config option, just to make it explicitly clear why an osd is getting skipped.

Done.